pymongo/network_layer.py

```diff
@@ -128,6 +178,136 @@ async def _async_sendall_ssl(
                 sent = 0
             total_sent += sent
 
+    async def _async_receive_ssl(
+        conn: _sslConn, length: int, dummy: AbstractEventLoop
+    ) -> memoryview:
+        mv = memoryview(bytearray(length))
+        total_read = 0
+        while total_read < length:
+            try:
+                read = conn.recv_into(mv[total_read:])
+            except BLOCKING_IO_ERRORS:
+                await asyncio.sleep(0.5)
```

Comment:
RIP windows perf. Can you schedule some of the windows tasks to see the impact?

For anyone else following along we'll fix this when we migrate to asyncio streams in: https://jira.mongodb.org/browse/PYTHON-4493

Reply:
This isn't a new change, it's already in master.

Reply:
Good point, I'll schedule some Windows tasks.